extensions/wikia/AdEngine/AdEngine2ExitstitialHooks.class.php

```diff
@@ -115,7 +115,7 @@ static private function getExitstitialUrlsWhiteList() {
 		}
 
 		$wikiDomains = WikiFactory::getDomains( $wgCityId );
-		if ( $wikiDomains !== false ) {
+		if ( is_array($wikiDomains) ) {
 			$whiteList = array_merge( $wikiDomains, $whiteList );
 		}
```

Comment:
Let's add Kibana logging here for a week to see what is the `$wgCityId` and non-array `$wikiDomains`.